cms/templates/maintenance/_force_publish_course.html

```diff
@@ -0,0 +1,80 @@
+<%page expression_filter="h"/>
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<div class="wrap-instructor-info studio-view maintenance-form">
+    <form class="form-create" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value="${csrf_token}"/>
+        <div class="wrapper-form">
+            <fieldset>
+                <ol class="list-input">
+                    <li class="field text required" id="field-course-name">
+                        <label for="course-id">${_('Course ID')}</label>
+                        <input id="course-id" type="text" name="course-id" aria-required="true"
+                               value="${form_data['course_id']}"
+                               placeholder="${_('e.g. edX+DemoX+Demo_Course')}"/>
+                        % if error:
+                        <div class="error">${msg}</div>
```

Comment:
To make this accessible, you should author it this way:
```
<!-- the error message container should be on the DOM at page load, even if empty. It should be referenced  on the input with aria-describedby -->
<input id="course-id" type="text" name="course-id" aria-describedby="cid-error" required value="${form_data['course_id']}"
  placeholder="${_('e.g. edX+DemoX+Demo_Course')}"/>
<!-- the error message container should have an aria-live=polite so that it reads the message to the user when updated-->
<div class="error" aria-live="polite" id="cid-error"> 
% if error: 
  <%  $msg %>
% endif
</div>
``` 

You should also move the focus to the form field (input) that has the first error.